modules/aws/README.md

```diff
@@ -17,6 +17,22 @@
     ]
     ```
 
+2. Upload a file:
```

Comment:
Could you also add an example for `uploadInBatch`?

Reply:
I would love to do that in the separate PR, because I would like to make a BC and rename the method to smth like `mergeJsonObjectsAndUploadAsOneFile` -> any idea for a shorter name?

The problem is it is not sending really in batch but as one file :-/